projects/elife.yaml

```diff
@@ -7,22 +7,26 @@ defaults:
     salt: "3003" # the version of Salt to install on ec2 instances
     terraform:
         # the version of Terraform this project requires
-        version: "0.11.15" # next version is "0.13.7" see `install-terraform.sh`
+        version: "0.13.7" # next version is "0.14.11" see `install-terraform.sh`
         # the version of Terraform providers (plugins) required.
         provider-aws:
+            source: "-/aws"
             version: "2.28.0"
         provider-tls:
+            source: "-/tls"
             version: "2.2"
         provider-fastly:
-            # source: fastly/fastly # provider changes from "-/fastly" (Hashicorp) for provider versions >1.0.0
-            # version: "1.1.4"
-            version: "0.9.0"
+            source: "fastly/fastly"
+            version: "1.1.4"
         provider-vault:
-            version: "1.3" # # '1.9.0' last version to support terraform 0.11.x
+            source: "-/vault"
+            version: "3.14.0"
         provider-google:
-            version: "1.20.0"
-        provider-eks:
-            version: "1.5.2"
+            source: "-/google"
+            version: "2.20.3"
+        provider-kubernetes:
```

Comment:
@scottaubrey , I've renamed `provider-eks` to `provider-kubernetes` fixing that bug we had before. Shouldn't affect anything, just an fyi.